### ISSUE TITLE
Can maxBytes use with S3 or GridFS?

### ISSUE THREAD

**Issue by faller** [balderdashy/skipper#73]
Is it only implement with skipper-disk ? thank you!

**Comment by tjwebb** [comment]
@faller did you ever get clarity on this?

**Comment by faller** [comment]
@tjwebb I turned to use protocal of Tus.io and implement myself

**Comment by sailsbot** [comment]
Thanks for posting, @faller.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by Clan-Utility** [comment]
For my test, maxBytes does not work for GridFS

**Comment by MuslimMunir** [comment]
for me maxBytes is not working for skipper s3 its crashing server
